google/ggrc-core
# GGRC-3743/GGRC-3745 Conflict resolution for Custom Attributes

# Issue description

Conflict resolution does not work for custom attributes.

# Steps to test the changes

1) Have at least 2 GCA for controls.
2) Create a control.
3) Open edit modal for the control in 2 separate tabs.
4) In 1st tab modify 1st GCA and click save.
5) In 2nd tab modify 2nd GCA and click save.
6) Check that both changes were saved.

# Solution description

1) Reimplement conflict resolving process to have ability to add special handlers for some fields.
2) Add conflict resolver for CA.

# Sanity checklist




# PR Review checklist
